x-pack/plugins/security_solution/cypress/tasks/search_bar.ts

```diff
@@ -24,15 +24,21 @@ export const openAddFilterPopover = () => {
   cy.get(GLOBAL_SEARCH_BAR_ADD_FILTER).click();
 };
 
-export const fillAddFilterForm = ({ key, value }: SearchBarFilter) => {
+export const fillAddFilterForm = ({ key, value, operator }: SearchBarFilter) => {
   cy.get(ADD_FILTER_FORM_FIELD_INPUT).should('exist');
   cy.get(ADD_FILTER_FORM_FIELD_INPUT).should('be.visible');
-  cy.get(ADD_FILTER_FORM_FIELD_INPUT).type(key);
+  cy.get(ADD_FILTER_FORM_FIELD_INPUT).type(`${key}{downarrow}`);
   cy.get(ADD_FILTER_FORM_FIELD_INPUT).click();
   cy.get(ADD_FILTER_FORM_FIELD_OPTION(key)).click({ force: true });
-  cy.get(ADD_FILTER_FORM_OPERATOR_FIELD).click();
-  cy.get(ADD_FILTER_FORM_OPERATOR_OPTION_IS).click();
-  cy.get(ADD_FILTER_FORM_FILTER_VALUE_INPUT).type(value);
+  if (!operator) {
```

Comment:
We are trying to avoid conditionals inside tasks methods.

Reply:
I understand avoiding conditional inside assertions as they may become in-deterministic but is there particular reason to avoid conditionals in tasks. 

Nonetheless, this condition can still be removed by making some changes but the next condition `if (value)` cannot be removed as filling the value in the filter form depends on the operator you have selected. Please see below screenshots for comparison.

|Value Field visible| Value Field not visible |
|--|--|
|![image](https://user-images.githubusercontent.com/7485038/200840069-b4011c03-6463-461b-ab9e-540d74cc2f50.png)| ![image](https://user-images.githubusercontent.com/7485038/200840169-541a4ddc-e020-4f1a-9cac-c3ccd36fe1be.png)|